hyppo/ksample/__init__.py

```diff
@@ -5,7 +5,8 @@
 from .ksamp import KSample
 from .manova import MANOVA
 from .mmd import MMD
-
+from .smoothCF import SmoothCFTest
+from .mean_embedding import MeanEmbeddingTest
 __all__ = [s for s in dir()]  # add imported tests to __all__
```

Comment:
bring back the space, part of black formatting

Reply:
Added blank line